checkov/terraform/checks_infra/solvers/complex_solvers/and_solver.py

```diff
@@ -7,15 +10,13 @@
 class AndSolver(BaseComplexSolver):
     operator = Operators.AND
 
-    def __init__(self, solvers, resource_types):
+    def __init__(self, solvers: List[BaseSolver], resource_types: List[str]) -> None:
         super().__init__(solvers, resource_types)
 
-    def _get_operation(self, *args):
+    def _get_operation(self, *args: Any) -> Any:
```

Comment:
@gruebel `Any` doesn't say much... What are the cases?

Reply:
the typing should be refined here for all the `_get_operation` methods, but to do that it would be great to actually. not use `*args` and `**kwargs` at all. I would recommend to do it in separate step.